content/en/graphing/faq/why-does-zooming-out-a-timeframe-also-smooth-out-my-graphs.md

```diff
@@ -1,20 +1,28 @@
 ---
 title: Why does zooming out a timeframe also smooth out my graphs?
 kind: faq
+disable_toc: true
+further_reading:
+- link: "/developers/metrics/types/"
+  tag: "Documentation"
+  text: "Discover Datadog metrics types"
+- link: "/graphing/functions/rollup/"
+  tag: "Documentation"
+  text: "Learn more about the rollup function"
 ---
-Within Datadog, a graph can only contain a set number of points and, as the timeframe over which a metric is viewed increases, aggregation between points occurs to stay below that set number. Thus you will lose in granularity as you increase the timeframe. For instance, for a 4-hour time window, data is aggregated to have one value per minute. As you "zoom out" (i.e. select a larger timeframe) the data shown on the graph will represent larger time period.
 
-You can append the [`.rollup()` function][1] to your query to adjust the method and granularity of time aggregation. Datadog rolls up data points automatically, based on the in-app metric type: `gauge` metrics are averaged, whereas `count` and `rate` metrics are summed. If you wanted to aggregate the sum of the metric over a one day period, you could append .rollup(sum, 86400) to your query. If you want to keep an eye on the max values, you may use the maximum aggregation .rollup(max).
+Within Datadog, a graph can only contain a set number of points and, as the timeframe over which a metric is viewed increases, aggregation between points occurs so that the number of points remains under that set number. Thus, granularity is lost as the timeframe increases. For instance, for a four hour time window, data is aggregated to have one value per minute for a line graph, and one value per two minutes for a bar graph. As you "zoom out" (i.e. select a larger timeframe) the data shown on the graph represents a longer time period.
 
-Here is a bar graph displaying a week's worth of cpu usage for a host without using the .rollup() function:
+{{< img src="graphing/faq/smooth_line.mp4" alt="Smoothing a line graph" responsive="true" video="true" width="90%" >}}
 
-{{< img src="graphing/faq/smooth_1.png" alt="smooth_1" responsive="true" >}}
+When bars are displayed the rollup interval is more obvious:
 
-And here is the same metric, graphed using a day-long rollup with .rollup(86400):
+{{< img src="graphing/faq/smoothing.mp4" alt="Smoothing a bar graph" responsive="true" video="true" width="90%" >}}
 
-{{< img src="graphing/faq/smooth_2.png" alt="smooth_2" responsive="true" >}}
+**Note**: You can manually append the `.rollup()` function to your query to adjust the method and granularity of time aggregation. Datadog rolls up data points automatically by default. Datadog averages values in the rollup interval for the `GAUGE` metric type. For `RATE` and `COUNT` metric types, the behavior differs, and values are summed without any interpolation. Learn more with the documentation about [in-application metric types modifiers][1].
```

Comment:
> Datadog averages values in the rollup interval for the `GAUGE` metric type. For `RATE` and `COUNT` metric types, the behavior differs, and values are summed without any interpolation. 

This needs to get fixed? RATE and COUNT use rollup(avg), except when the UI adds .as_count() - however, other kinds of queries exist. .as_count() is not added when requests are made by the API, from monitors, etc. We should be explicit about the actual behavior.